server/views/pages/video.html

```diff
@@ -60,6 +60,7 @@ <h1 id="title" class="govuk-hub-media-player__title govuk-heading-xl govuk-body
         controls
         preload="auto"
         poster="{{ data.image.url }}"
+        style="max-height: 500px;"
```

Comment:
I wonder if this could live as a CSS class rather than inline? 

Would it be worth us using `vh` to set this as a percentage of the height of the viewport to make this more flexible than a pixel count?

Reply:
@spikeheap I should've started as a Draft anyways 🤦 It doesn't work anyway.